src/pages/statistics/index.tsx

```diff
@@ -1,12 +1,235 @@
-// import React from 'react'
+import React, { useEffect, useState } from 'react'
+import {
+    Box,
+    Typography,
+    Drawer,
+    List,
+    ListItemButton,
+    ListItemText,
+    IconButton,
+    Accordion,
+    AccordionSummary,
+    AccordionDetails,
+    styled,
+} from '@mui/material'
+import MenuIcon from '@mui/icons-material/Menu'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import NavBar from '../../components/NavBar'
+import Chart from '../../components/Chart'
+import StatisticsTable from '../../components/StatisticsTable'
+import { data } from '../../data/statisticsData'
 import './statistics.css'
+import Footer from '../../components/Footer'
+
+const ListButton = styled(ListItemButton)({
+    '&.Mui-selected': {
+        backgroundColor: '#007FFF',
+        '&:hover': {
+            backgroundColor: '#005BBB',
+        },
+    },
+    '&:hover': {
+        backgroundColor: '#CCCCCC',
+    },
+})
+
+const ListText = styled(ListItemText)({
+    color: 'white',
+    fontFamily: 'Georgia',
+    fontWeight: 'bold',
+})
+
+const drawerWidthOpen = '20vw'
+const drawerWidthClosed = '4vw'
+
+const Statistics: React.FC = () => {
+    const [sidebarOpen, setSidebarOpen] = useState(true)
+    const [selectedChart, setSelectedChart] = useState<string | null>('plugins')
+    const [selectedTab, setSelectedTab] = useState<string | null>('overall')
+    const [selectedYear, setSelectedYear] = useState<number | null>(null)
+
+    const toggleSidebar = () => {
+        setSidebarOpen(!sidebarOpen)
+    }
+
+    const handleChartSelect = (chart: string) => {
+        setSelectedChart(chart)
+        setSelectedTab('overall')
+        setSelectedYear(null) // deselect any year when selecting a chart
+        if (window.innerWidth < 600) {
+            setSidebarOpen(false)
+        }
+    }
+
+    const handleYearSelect = (year: number | null) => {
+        setSelectedYear(year)
+        setSelectedTab('monthly')
+        setSelectedChart(null) // deselect any chart when selecting a year
+    }
+
+    useEffect(() => {
+        setSelectedChart('plugins')
+        setSelectedTab('overall')
+    }, [])
 
-export default function statistics() {
     return (
-        <div className="top">
-            Page Under Construction
-            <h1 className="sta">Statistics</h1>
-            <p>test</p>
-        </div>
+        <Box className="background" sx={{ width: '100vw', height: '100vh', display: 'flex', flexDirection: 'column' }}>
```

Comment:
```suggestion
        <Box className="background" sx={{ width: '100vw', minHeight: '100vh', display: 'block'}}>
```

Reply:
This suggestion is mostly for setting a min-height instead of a height for the statistics index.html page, and also to use block display so that the footer would drop below the viewport.

Reply:
Kris - this change causes issues. It causes the footer to move up inbstead of sticking to the bottom. Are we sure this is necessary? Also, it was a stylistic choice to have the footer in the stats page, but we can remove it alltogether if you guys think it would look better

Reply:
Using minHeight also causes the footer to be pushed out of view when opening monthly analysis accordion.

Reply:
I think it has to do with the screen size you have. Your current CSS settings is causing problems with a regular laptop, hence my suggestion. But we should definitely get rid of the footer for the "Statistics in detail" page since we don't really need it.